### DISCUSSION TITLE
Creating custom tracks from coordinates, export coordinates to fasta

### DISCUSSION THREAD

**Discussion by cganote** [GMOD/jbrowse-components#3997] [Q&A]
Hi Jbrowse community,

I have a few things I'd like to do quickly. I'm using JBrowse from withing Galaxy EU, so I don't know if I have a lot of flexibility - maybe these features are available on newer releases or plugins - but I'd like to be able to get arbitrary coordinates from my reference sequence into my clipboard in fasta format, or maybe create a custom track so I can copy the sequence from there. Any easy way to do that? Say, chr1 from 542 to position 600?

Thanks for any hints!
-Carrie

**Comment by cmdcolin** [comment]
this isn't particularly user friendly, and only works if the Galaxy EU let's you manipulate URL parameters, but...you could hand-edit the URL parameters to produce a 'session track' with coordinates at your desired location

example creating a feature at 100-200

https://jbrowse.org/code/jb2/main/?config=test_data/volvox/config.json&loc=ctgA:1-800&assembly=volvox&tracks=gff3tabix_genes,volvox_filtered_vcf,volvox_microarray,volvox_cram,url_track&sessionTracks=[{"type":"FeatureTrack","trackId":"url_track","name":"URL%20track","assemblyNames":["volvox"],"adapter":{"type":"FromConfigAdapter","features":[{"uniqueId":"one","refName":"ctgA","start":100,"end":200,"name":"My%20feature"}]}}]

docs for this here https://jbrowse.org/jb2/docs/urlparams/

**Reply by cmdcolin** [reply]
it could be good for us to make an in-app feature for this though! I am not sure exactly how it would look for the user (a similar feature in jbrowse 1 allowed users to manually specify the coordinates of highlighted region, and then get sequence for the highlighted region) but jbrowse 2 doesn't have that function yet
